frontend/src/components/RFI/NMF.vue

```diff
@@ -151,19 +125,22 @@
 
 <script>
 import { mapActions, mapState } from 'pinia';
-import { useAppStore } from '../../store/app.js';
-import { useApplicationStore } from '../../store/application.js';
-import { useNmfAppStore } from '../../store/nmfApp.js';
-import { useNavBarStore } from '../../store/navBar.js';
 
-import alertMixin from '../../mixins/alertMixin.js';
-import NavButton from '../../components/util/NavButton.vue';
-import FacilityHeader from '../guiComponents/FacilityHeader.vue';
+import NavButton from '@/components/util/NavButton.vue';
+import FacilityHeader from '@/components/guiComponents/FacilityHeader.vue';
+
+import { useAppStore } from '@/store/app.js';
+import { useApplicationStore } from '@/store/application.js';
+import { useNmfAppStore } from '@/store/nmfApp.js';
+import { useNavBarStore } from '@/store/navBar.js';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import rules from '@/utils/rules.js';
 
 let model = { x: [] };
 
 export default {
-  name: 'CcfriRequestMoreInfo',
+  name: 'NmfRequestMoreInfo',
```

Comment:
In general we use the file name as the component name. But if this is consistent with CcfriRequestMoreInfo then it's probably ok. Also this component has a very generic name.